app/auto-eligibility/config/index.js

```diff
@@ -0,0 +1,26 @@
+const Joi = require('joi')
+
+const schema = Joi.object({
+  emailTemplates: {
+    waitingList: Joi.string().uuid(),
+    genericIneligible: Joi.string().uuid(),
+    applyServiceInvite: Joi.string().uuid()
+  }
+})
+
+const config = {
+  emailTemplates: {
+    waitingList: process.env.NOTIFY_TEMPLATE_ID_WAITING_LIST,
```

Comment:
NOTIFY_EMAIL_TEMPLATE_ID_WAITING_LIST?

I know that NOTIFY suggests it's an "email template" but then it's a bit inconsistent with the property name "emailTemplates" where the word notify is not used.